### PR TITLE
Lk star sf tag

### PR DESCRIPTION
**Pull request by ekiernan**
This PR adds new dockers for Optimus metric calculation and h5ad generation. The pipeline now calculates exonic and intronic metrics using STARsolo v2.7.11a with the new sF tag in the BAM. The metric CSV files were additionally modified to no longer remove columns.

----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [ ] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ ] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

**Review by kevinpalis — APPROVED** [review]
Looks good!

**Comment by ekiernan** [comment]
Looked at all SmartTest results and documented validation here: https://docs.google.com/document/d/15XPzcZNygfLR1Vz6PWJ2vE4nyJtrTRFp1lfU1e5NbQU/edit.

All failures are a result of the new columns added from the BAM sF tag for Multiome, Optimus, and Slideseq.

Will update truth

**Comment by ekiernan** [comment]
Truth for Optimus: https://gotc-jenkins.dsp-techops.broadinstitute.org/job/warp-workflow-tests/53163/

Truth for Multiome: https://gotc-jenkins.dsp-techops.broadinstitute.org/job/warp-workflow-tests/53164/

Truth for Slideseq: https://gotc-jenkins.dsp-techops.broadinstitute.org/job/warp-workflow-tests/53165/

File: website/docs/Pipelines/Optimus_Pipeline/Bam_tags.md [hunk 1]
```diff
@@ -27,3 +27,4 @@ The Optimus Pipeline outputs a barcoded BAM file of aligned reads. There are mul
 | jI | [jI:B:I,Start1,End1,Start2,End2,...] Start and End of introns for all junctions (1-based) | [STAR](https://github.com/alexdobin/STAR) |
 | jM | [jM:B:c,M1,M2,...] Intron motifs for all junctions (i.e. N in CIGAR)| [STAR](https://github.com/alexdobin/STAR) |
 | nM | The number of mismatches per (paired) alignment | [STAR](https://github.com/alexdobin/STAR) |
+| sF |Tthe feature type and number of genes for each read | [STAR v2.7.11a and later](https://github.com/alexdobin/STAR/releases)
```
Comment (nikellepetrillo) [inline website/docs/Pipelines/Optimus_Pipeline/Bam_tags.md:30]:
```suggestion
| sF |The feature type and number of genes for each read | [STAR v2.7.11a and later](https://github.com/alexdobin/STAR/releases)
```

File: pipelines/skylab/optimus/Optimus.wdl [hunk 2]
```diff
@@ -67,7 +67,7 @@ workflow Optimus {
 
   # version of this pipeline
 
-  String pipeline_version = "5.8.4"
+  String pipeline_version = "6.0.0"
```
Comment (jessicaway) [inline pipelines/skylab/optimus/Optimus.wdl:70]: 6.0.0 wow! That's exciting!

File: tasks/skylab/Metrics.wdl [hunk 3]
```diff
@@ -9,7 +9,7 @@ task CalculateCellMetrics {
 
     # runtime values
 
-    String docker = "us.gcr.io/broad-gotc-prod/warp-tools:1.0.1-1679941323"
+    String docker = "us.gcr.io/broad-gotc-prod/warp-tools:1.0.6-1692962087"
```
Comment (nikellepetrillo) [inline tasks/skylab/Metrics.wdl:12]: i noticed this docker tag is different than the other warp-tools tags (1.0.5-1692706846). I just want to make sure this was on purpose?

File: pipelines/skylab/slideseq/SlideSeq.changelog.md [hunk 4]
```diff
@@ -1,3 +1,11 @@
+# 2.0.0
+2023-08-22 (Date of Last Commit)
+
+* Updated Optimus and Slideseq pipelines to include STARsolo v2.7.11a
```
Comment (jessicaway) [inline pipelines/skylab/slideseq/SlideSeq.changelog.md:4]:
```suggestion
* Updated Slideseq pipeline to include STARsolo v2.7.11a
```

File: website/docs/Pipelines/Optimus_Pipeline/Bam_tags.md (hunk 1 above)
Comment (jessicaway) [reply website/docs/Pipelines/Optimus_Pipeline/Bam_tags.md:30]: May want to include the preceding space as well to match the other entries 
```suggestion
| sF | The feature type and number of genes for each read | [STAR v2.7.11a and later](https://github.com/alexdobin/STAR/releases)
```

**Review by jessicaway — APPROVED** [review]

File: tasks/skylab/Metrics.wdl (hunk 3 above)
Comment (ekiernan) [reply tasks/skylab/Metrics.wdl:12]: I missed that you were asking about 1.05 in the h5ad task. Thanks for catching! I updated and am retesting now.

File: pipelines/skylab/optimus/Optimus.wdl (hunk 2 above)
Comment (ekiernan) [reply pipelines/skylab/optimus/Optimus.wdl:70]: Very exciting!

**Pull request merged**